docs/getting_started/user-guide.rst

```diff
@@ -1114,19 +1114,87 @@ Add
+tags will also get updated automatically. 
+
+Add a Local Container
+^^^^^^^^^^^^^^^^^^^^^
+
+As an example, let's start with the container ``salad_latest.sif``. We have it
+on our local machine and cannot pull it from a registry. First, let's run ``shpc add``
+and tell shpc that we want it under the ``dinosaur/salad`` namespace.
+
+.. code-block:: console
+
+    $ shpc add salad_latest.sif dinosaur/salad:latest
+    Registry entry dinosaur/salad:latest was added! Before shpc install, edit:
+    /home/vanessa/Desktop/Code/shpc/registry/dinosaur/salad/container.yaml
+
+At this point, you should open up the container.yaml generated and edit to your liking.
+This usually means updating the description, maintainer, aliases, and possibly providing a url
+to find more information or support. Also notice we've provided the tag to be latest. If you update this registry
+entry in the future with a new version, you'll want to provide a new tag. If you provide
+an existing tag, you'll be asked to confirm before continuing. When you are happy, 
+it's time to install it, just as you would a regular container!
 
 .. code-block:: console
 
-    $ shpc add salad_latest.sif vanessa/salad:latest
+    $ shpc install dinosaur/salad:latest
+
+
+And this will generate the expected module and container in your respective directory bases:
+
+
+.. code-block:: console
+
+    $ tree modules/dinosaur/salad/
+    modules/dinosaur/salad/
+    └── latest
+        ├── 99-shpc.sh
+        └── module.lua
+
+    1 directory, 2 files
+
+    $ tree containers/dinosaur/salad/
+    containers/dinosaur/salad/
+    └── latest
+        └── sha256:77c7326e74d0e8b46d4e50d99e848fc950ed047babd60203e17449f5df8f39d4.sif
+
+    1 directory, 1 file
+
+
+Add a Registry Container
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Let's say we want to generate a container.yaml recipe for a container on Docker Hub.
+Let's say we want to add `vanessa/pokemon <https://hub.docker.com/r/vanessa/pokemon>`_.
+First, let's run ``shpc add``. Note that we provide the ``docker://`` unique resource
+identifier to tell shpc it's from a Docker (OCI) registry.
+
+.. code-block:: console
+
+    $ shpc add docker://vanessa/pokemon
+    Registry entry vanessa/pokemon:latest was added! Before shpc install, edit:
+    /home/vanessa/Desktop/Code/shpc/registry/vanessa/pokemon/container.yaml
+
+
+And that's it! The container module will use the same namespace, ``vanessa/pokemon`` as the Docker image,
+and we do this purposefully as a design decision. Note that ``add`` previously would add the container directly to the module
+directory, and as of version 0.0.49 it's been updated to generate the container.yaml first.
+Also note that ``add`` is only supported for Singularity, as Docker and Podman containers are 
+typically provided via registries. If you are looking for support for add for another
+container technology, please `open a new issue <https://github.com/singularityhub/singularity-hpc/issues>`_.
```

Comment:
Hi @vsoch . This seems to contradict the lines above 🤔 . Is Docker supported ?

Reply:
The old add didn't support docker/podman - e.g., the old add would say "take this .sif on my filesystem and manually move it into shpc." The new add (which generates a container.yaml for you) indeed has support!